src/js/stores/MesosStateStore.js

```diff
@@ -81,6 +81,7 @@ class MesosStateStore extends GetSetBaseStore {
       .distinctUntilChanged()
       .map(message => parsers(this.getLastMesosState(), JSON.parse(message)))
       .do(state => {
+        // The tasks received here have not GPU
```

Comment:
@nLight For some reason the tasks coming form Mesos don't have GPU, but only CPU and Mem. Is this something we can fix?

Reply:
From the UI point of view if Mesos provides the info it should be there. Can we make sure that mesos provides the info? Unless it does we need an urgent ticket for the Mesos team 💃

Reply:
Could you please remove the comment?